studiopress/genesis-sample
# Hide featued image on imported content

**Summary of change:**
Hides the featured image on home, about, and landing pages if the images are enabled site-wide and a featured image is set.

**This PR has been:**

**Have the changes in this PR been added to the documentation for this project?**
Does not apply.

**How to test:**

1. Use this branch with Genesis 3.1 and install a starter pack.
2. Go to Appearance > Customize > Theme Settings > Singular Content and enable featured images on pages.
3. Set a featured image on home, about, and landing pages.
4. Make sure the featured image does not show on these pages.

**Suggested Changelog Entry:**
Changed: One-click theme setup to include meta for new Genesis 3.1 features.
